### PR TITLE
Add styles for tiling video

### PR DESCRIPTION
**Pull request by huijing**
This is not the optimal solution because a more robust solution would adjust the styles depending on the number of streams but for now this is the limited implementation that sort of works with 4 to 9 streams.
Someone please figure out the math formula for tiling...

### CODE REVIEW

File: app/javascript/packs/party.js [hunk 1]
```diff
@@ -49,6 +50,18 @@ export default class Party {
...
+
+      // This function runs whenever a client connects to a session
+      connectionCreated: function(event) {
+        self.connectionCount++;
+        streamLayout(self.subscribers, self.connectionCount);
+      },
+
+      // This function runs whenever a client disconnects from the session
+      connectionDestroyed: function(event) {
+        self.connectionCount++;
```
Comment (benhayehudi) [inline app/javascript/packs/party.js:63]: Should this be `self.connectionCount--` when the connection has been destroyed?

**Comment by benhayehudi** [comment]
@huijing, when I hit the threshold to go to grid mode, this is what it starts to look like

Screen Shot 2020-04-23 at 19 37 08

**Comment by huijing** [comment]
> 
> 
> @huijing, when I hit the threshold to go to grid mode, this is what it starts to look like
> 
> Screen Shot 2020-04-23 at 19 37 08

Yeah, that's the expected behaviour, we could add some sort of nicer background other than white to camouflage the fact.

**Comment by benhayehudi** [comment]
Looks great, thank you so much @huijing!

**Pull request merged**